modules/pubmaticBidAdapter.js

```diff
@@ -71,25 +71,25 @@ function _parseAdSlot(bid) {
   bid.params.adSlot = _cleanSlot(bid.params.adSlot);
 
   var slot = bid.params.adSlot;
-  var splits = slot.split(':');
+  try{
+    var splits = slot.split('@');
 
-  slot = splits[0];
-  if (splits.length == 2) {
-    bid.params.adUnitIndex = splits[1];
-  }
-  splits = slot.split('@');
-  if (splits.length != 2) {
-    utils.logWarn('AdSlot Error: adSlot not in required format');
-    return;
+    slot = splits[0];
+    if (splits.length == 2) {
+      bid.params.adUnitIndex = splits[1].split(":")[1];
```

Comment:
This line sets adUnitIndex to undefined in case of bid = {params:{adSlot: 'abcd@10x80'}};
Default value for adUnitIndex is '0'